### DISCUSSION TITLE
Show full description Referenties / References

### DISCUSSION THREAD

**Discussion by GerlofdeHaan** [alkem-io/alkemio#577] [Ideas] [answered]
![Referenties](https://user-images.githubusercontent.com/102361398/162709581-d7a66619-2d67-4d97-8ee2-f2cecf5a6b7d.JPG)

The description of References is now cut off at x characters (tooltip shows full description).

**Comment by techsmyth** [comment, marked as answer]
This actually went in the release on 11 April!
